src/libcore/num/mod.rs

```diff
@@ -187,13 +187,29 @@ macro_rules! int_impl {
      $sub_with_overflow:path,
      $mul_with_overflow:path) => {
         /// Returns the smallest value that can be represented by this integer type.
+        ///
+        /// # Examples
+        ///
+        /// Basic usage:
```

Comment:
I would rather not add more, as I'm gonna have to go back and remove them soon, and this just creates more work :smile: